### PR TITLE
bugfix/AP5-73

### PR DESCRIPTION
**Pull request by jasonephraim**
fix main window overlap issue and corrected nav banner so the panel borders match color

### CODE REVIEW

File: rigex-front-end/src/components/navBar/navBar.scss [hunk 1]
```diff
@@ -1,34 +1,33 @@
 @import '../../styles/colors.scss';
 
 .iconBox {
-    display: "flex";
-    flex-direction: "row";
-    justify-content: "space-between";
-    width: 120px;
-    margin-left: auto;
+  display: 'flex';
+  flex-direction: 'row';
```
Comment (BarretteBanner) [inline rigex-front-end/src/components/navBar/navBar.scss]: Why do we have quotes on these

File: rigex-front-end/src/pages/Landing/Landing.js [hunk 2]
```diff
@@ -64,6 +64,8 @@ const Landing = () => {
     </div>
   );
 
+  console.log(listOfRigs);
```
Comment (BarretteBanner) [inline rigex-front-end/src/pages/Landing/Landing.js]: 👀

File: rigex-front-end/src/components/navBar/navBar.scss (hunk 1 above)
Comment (jasonephraim) [reply rigex-front-end/src/components/navBar/navBar.scss]: fixed

File: rigex-front-end/src/pages/Landing/Landing.js (hunk 2 above)
Comment (jasonephraim) [reply rigex-front-end/src/pages/Landing/Landing.js]: removed

**Review by BarretteBanner — APPROVED** [review]

**Review by mmevans — APPROVED** [review]

**Pull request merged**